src/extensions/default/JavaScriptCodeHints/ScopeManager.js

```diff
@@ -1086,23 +1099,20 @@ define(function (require, exports, module) {
                         handleTimedOut(response);
                     } else if (type === MessageIds.TERN_WORKER_READY) {
                         moduleDeferred.resolveWith(null, [_ternNodeDomain]);
+                    } else if (type === "ReInitTern") {
```

Comment:
This should be a const like the other ones (`MessageIds.RE_INIT_TERN`)

Reply:
Wait, I just saw that this only gets called if `!messageIds`, so maybe just `RE_INIT_TERN`

Reply:
Fixed